ui/src/components/rollouts/QuickEditRolloutForm.tsx

```diff
@@ -0,0 +1,224 @@
+                  <label
+                    htmlFor="percentage"
+                    className="mb-2 block text-sm font-medium text-gray-900"
+                  >
+                    Percentage
+                  </label>
+                  <Input
+                    id="percentage-slider"
+                    name="percentage"
+                    type="range"
+                    className="hidden h-2 w-full cursor-pointer appearance-none self-center rounded-lg align-middle bg-gray-200 dark:bg-gray-700 sm:block"
+                  />
+                  <div className="relative">
+                    <div className="pointer-events-none absolute inset-y-0 left-0 flex items-center pl-3">
+                      %
+                    </div>
+                    <Input
+                      type="number"
+                      id="percentage"
+                      max={100}
+                      min={0}
+                      name="percentage"
+                      className="pl-10 text-center"
+                    />
+                  </div>
+                </div>
+              ) : (
+                <div className="space-y-1 px-4 sm:grid sm:grid-cols-3 sm:gap-4 sm:space-y-0 sm:p-2">
+                  <div>
+                    <label
+                      htmlFor="segmentKey"
+                      className="block text-sm font-medium text-gray-900 sm:mt-px sm:pt-2"
+                    >
+                      Segment
+                    </label>
+                  </div>
+                  <div className="sm:col-span-2">
+                    <Combobox<FilterableSegment>
+                      id="segmentKey"
+                      name="segmentKey"
+                      placeholder="Select or search for a segment"
+                      values={segments.map((s) => ({
+                        ...s,
+                        filterValue: truncateKey(s.key),
+                        displayValue: s.name
+                      }))}
+                      selected={selectedSegment}
+                      setSelected={setSelectedSegment}
+                    />
+                  </div>
+                </div>
+              )}
+              <div className="space-y-1 px-4 sm:grid sm:grid-cols-3 sm:gap-4 sm:space-y-0 sm:p-2">
+                <label
+                  htmlFor="value"
+                  className="mb-2 block text-sm font-medium text-gray-900"
+                >
+                  Value
+                </label>
+                <Select
+                  id="value"
+                  name="value"
+                  value={formik.values.value}
+                  options={[
+                    { label: 'True', value: 'true' },
+                    { label: 'False', value: 'false' }
+                  ]}
+                  className="w-full cursor-pointer appearance-none self-center rounded-lg py-1 align-middle bg-gray-50"
+                />
```

Comment:
Take it or leave it: How about making this a toggle? Do you think that might be confusing?

Im probably asking for something that is a pain to make. I found this Flowbite thing, that I remember seeing someone mention on your twitter and I guess this isn't something you can just steal:
https://flowbite.com/docs/forms/toggle/

Reply:
to me a toggle represents 'on/off' and not really true / false as boolean values.. idk if that makes sense.. but i kind of still think a select makes the most sense here